File: example-nonblocking/src/main/java/com/davin111/examplenonblocking/Content.java
package com.davin111.examplenonblocking;

import lombok.Getter;
import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Table;

@Table("content")
@Getter
public class Content {
    @Id
    private Long id;

    private String name;

    private Boolean isPremium;

    private String imageUrl;

    private String videoUrl;
}

File: example-nonblocking-kotlin/src/main/kotlin/com/davin111/examplenonblockingkotlin/Content.kt
package com.davin111.examplenonblockingkotlin

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Table("content")
class Content(
    @Id
    val id: Long,

    val name: String,

    val isPremium: Boolean,

    val imageUrl: String?,

    val videoUrl: String?,
)
